ereuse_tag/view.py

```diff
@@ -41,3 +44,11 @@ def post(self):
         response = jsonify(ids)
         response.status_code = 201
         return response
+
+
+class VersionView(View):
+    def get(self, *args, **kwargs):
+        """Get version."""
+
+        v = "{}".format(__version__)
```

Comment:
I think that this line does nothing and could be safely removed, couldn't?

Reply:
It's true. I removed this line